static/scripts/autologout.js

```diff
@@ -0,0 +1,97 @@
+$(document).ready(() => {
+	const $autoLoggoutAlertModal = $('.auto-logout-alert-modal');
+	const rstDefault = 7200; // default remaining session time in sec
+	const showModalOnRemainingSeconds = $autoLoggoutAlertModal.find('.form-group').data('showOnRemainingSec') || 3600;
+	const maxTotalRetrys = 1;
+
+	let rst = rstDefault; // remaining session time in sec
+	let processing = false;
+
+	populateModalForm($autoLoggoutAlertModal, {});
+
+	const showAutoLogoutModal = ((status) => {
+		if (status === 'error') {
+			$autoLoggoutAlertModal.find('.sloth-default').hide();
+			$autoLoggoutAlertModal.find('.sloth-error').show();
+		} else {
+			$autoLoggoutAlertModal.find('.sloth-default').show();
```

Comment:
what is status here `default`? and after the if?

Reply:
would be nice to add a comment. I had also problems to understand.